File: Linked List/C++/GFG - IS LINKED LIST LENGTH EVEN.cpp
// GFG - Linked List Length Even or Odd
// QUESTION LINK - https://www.geeksforgeeks.org/problems/linked-list-length-even-or-odd/1

bool isLengthEven(struct Node **head) {
    Node *temp = *head;
    int count = 0;
    while (temp != NULL) {
        count++;
        temp = temp->next;
    }
    return (count % 2 == 0);
}

File: Linked List/Java/GFG - IS LINKED LIST LENGTH EVEN.java
// GFG - Linked List Length Even or Odd
// QUESTION LINK - https://www.geeksforgeeks.org/problems/linked-list-length-even-or-odd/1

class Solution {
    public boolean isLengthEven(Node head) {
        int count = 0;
        Node temp = head;
        while (temp != null) {
            count++;
            temp = temp.next;
        }
        return count % 2 == 0;
    }
}
